content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/250-querying-the-database.mdx

```diff
@@ -0,0 +1,419 @@
+      name: 'Rich',
+      email: 'hello@prisma.com',
+      posts: {
+        create: {
+          title: 'My first post',
+          body: 'Lots of really interesting stuff',
+          slug: 'my-first-post',
+        }
+      }
+    }
+  })
+
+  const allUsers = await prisma.user.findMany({
+    include: {
+      posts: true,
+    },
+  })
+  console.dir(allUsers, { depth: null })
+}
+```
+
+</SwitchTech>
+
+This code creates a new `User` record together with a new `Post` using a [nested write](../../../../../concepts/components/prisma-client/relation-queries#nested-writes) query. The `User` record is connected to the other one via the `Post.author` ↔ `User.posts` [relation fields](../../../../../concepts/components/prisma-schema/relations#relation-fields) respectively.
+
+Notice that you're passing the [`include`](../../../../concepts/components/prisma-client/select-fields#include-relations-and-select-relation-fields) option to `findMany` which tells Prisma Client to include the `posts` relations on the returned `User` objects.
+
+Run the code with this command:
+
+<SwitchTech technologies={['typescript', '*']}>
+
+```terminal copy
+npx ts-node index.ts
+```
+
+</SwitchTech>
+
+<SwitchTech technologies={['node', '*']}>
+
+```terminal copy
+node index.js
+```
+
+</SwitchTech>
+
+The output should look similar to this:
+
+```js no-lines
+[
+  {
+    id: '60cc9b0e001e3bfd00a6eddf',
+    email: 'hello@prisma.com',
+    name: 'Rich',
+    posts: [
+      {
+        id: '60cc9bad005059d6007f45dd',
+        slug: 'my-first-post',
+        title: 'My first post',
+        body: 'Lots of really interesting stuff',
+        userId: '60cc9b0e001e3bfd00a6eddf'
+      }
+    ]
+  }
+]
+```
+
+<SwitchTech technologies={['typescript']}>
+
+Also note that `allUsers` is _statically typed_ thanks to [Prisma Client's generated types](../../../../../concepts/components/prisma-client/advanced-type-safety/operating-against-partial-structures-of-model-types). You can observe the type by hovering over the `allUsers` variable in your editor. It should be typed as follows:
+
+```ts no-lines
+const allUsers: (User & {
+  posts: Post[]
+})[]
+
+export type Post = {
+  id: number
+  title: string
+  body: string | null
+  published: boolean
+  authorId: number | null
+}
+```
+
+</SwitchTech>
+
+The query added new records to the `User` and the `Post` tables:
+
+**User**
+
+| **id** | **email**           | **name**  |
+| :----- | :------------------ | :-------- |
+| `60cc9b0e001e3bfd00a6eddf`    | `"hello@prisma.com"` | `"Rich"` |
+
+**Post**
+
+| **id**                      | **createdAt**              | **title**          | **content** | **published** | **authorId** |
+| :-------------------------- | :------------------------- | :----------------- | :---------- | :------------ | :----------- |
+| `60cc9bad005059d6007f45dd`  | `2020-03-21T16:45:01.246Z` | `"My first post"`  | `Lots of really interesting stuff`      | `false`       | `60cc9b0e001e3bfd00a6eddf`          |
```

Comment:
Is the naming of the `id` column correct here or is is `_id` under the hood?
(not sure if we want to show the "under the hood" or the "Prisma level" here though)

Reply:
Changed to _id and removed references to table/ column